pwsh-extension-dev.code-workspace

```diff
@@ -0,0 +1,493 @@
+        "options": [
+          "Restore",
+          "Clean",
+          "Build",
+          "Test",
+          "Package"
+        ],
+        "default": "Clean"
+      },
+      {
+        "type": "pickString",
+        "id": "serverBuildCommand",
+        "description": "Which Invoke-Build Server Task?",
+        "options": [
+          "SetupDotNet",
+          "BinClean",
+          "Clean",
+          "Build",
+          "Test",
+          "TestPS74",
+          "TestE2EPwsh",
+          "TestPS51",
+          "TestE2EPowerShell",
+        ],
+        "default": "Clean"
+      }
+    ],
+
+  },
+  "launch": {
+    "version": "0.2.0",
+    "compounds": [
+      {
+        "name": "Launch Extension",
+        "configurations": [
+          "Launch",
+          "PowerShell Editor Services"
+        ],
+        "preLaunchTask": "Build",
+        "stopAll": true,
+        "presentation": {
+          "hidden": false,
+          "group": "Test",
+          "order": 1
+        }
+      },
+      {
+        "name": "Launch Extension - Temp Profile",
+        "configurations": [
+          "Launch-Temp",
+          "PowerShell Editor Services"
+        ],
+        "preLaunchTask": "PreLaunch",
+        "stopAll": true,
+        "presentation": {
+          "hidden": false,
+          "group": "Test",
+          "order": 2
+        }
+      },
+      {
+        "name": "Launch Extension - Isolated Profile",
+        "configurations": [
+          "Launch-Isolated",
+          "PowerShell Editor Services"
+        ],
+        "preLaunchTask": "Build",
+        "stopAll": true,
+        "presentation": {
+          "hidden": false,
+          "group": "Test",
+          "order": 3
+        }
+      },
+      {
+        "name": "Interactively Test Rename",
+        "configurations": [
+          "Launch Extension - Rename Test Cases",
+          "Attach to Editor Services"
+        ],
+        "preLaunchTask": "Build",
+        "stopAll": true,
+        "presentation": {
+          "hidden": false,
+          "group": "Test",
+          "order": 4
+        }
+      }
+    ],
+    "configurations": [
+      {
+        "name": "Launch",
+        "type": "extensionHost",
+        "request": "launch",
+        "env": {
+          "VSCODE_PARENT_SESSION_ID": "${command:GetVsCodeSessionId}",
+        },
+        "runtimeExecutable": "${execPath}",
+        "args": [
+          "--extensionDevelopmentPath=${workspaceFolder:Client}",
+          "${workspaceFolder:Client}/examples"
+        ],
+        "sourceMaps": true,
+        // This speeds up source map detection and makes smartStep work correctly
+        "outFiles": [
+          "${workspaceFolder:Client}/**/*.js",
+          "!**/node_modules/**",
+          "!**/.vscode-test/**"
+        ],
+        "skipFiles": [
+          "<node_internals>/**",
+          "**/node_modules/**",
+          "**/.vscode-test/**",
+          "**/app/out/vs/**" //Skips Extension Host internals
```

Comment:
```suggestion
          "**/app/out/vs/**", // Skips Extension Host internals
```

I have a feeling that this has been missing 😓

Reply:
Stepping into that extensionhost accidentally drives me crazy, as far as I know they don't publish the source maps.

Reply:
It's made debugging SO annoying. Thanks for the fix.